### DISCUSSION TITLE
Piper Tray (Windows system tray utility)

### DISCUSSION THREAD

**Discussion by jame25** [rhasspy/piper#441] [Show and tell]
To show my appreciation for Piper TTS, I have developed a small Windows tray utility written in C#. Feedback welcome.

<p align="center"> <img width="256" height="256" src="https://github.com/jame25/Piper-Tray/assets/13631646/95cf62c7-9248-4195-864b-57771d922bd6"></p>

[https://github.com/jame25/Piper-Tray](https://github.com/jame25/Piper-Tray)

## Features:

* Reads clipboard contents aloud
* Enable / Disable clipboard monitoring
* Many voices to choose from
* Change Piper TTS voice model
* Control Piper TTS speech rate
* Hotkey support (ALT + Q) to stop speech
* Prevent keywords being read (ignore.dict)

**Comment by W1Real** [comment]
Do you have SAPI5 integration, do you have plans for it? See [Piper TTS as a windows integration](https://github.com/rhasspy/piper/discussions/577) ->  #577 

Personally, I wanted to use it with Calibre reader. I have some old Ivona voices for that, they use SAPI5 integration.

**Reply by jame25** [reply]
SAPI 5 intergration would be a seperate project. I would use it myself with Calibre reader and I will look into it.

**Reply by jame25** [reply]
FYI, [Calibre reader](https://calibre-ebook.com/whats-new) added support for Piper TTS with the latest version (v7.18).
